diff.py

```diff
@@ -1780,8 +1782,100 @@ def process_reloc(self, row: str, prev: str) -> Tuple[str, Optional[str]]:
     def is_end_of_function(self, mnemonic: str, args: str) -> bool:
         return mnemonic == "blr"
 
+# Example: "cmp r0, #0x10"
+ARM32_COMPARE_IMM_PATTERN = r"cmp\s+(r[0-9]|1[0-3]),\s+#(\w+)"
+
+# Example: "add pc, r1"
+ARM32_JUMP_TABLE_START = r"add\s+pc,\s*r"
+
+# Examples:
+#  - "44:   00060032    .word   0x00060032"
+#  - "48:   0032        .short  0x0032"
+#  - "4a:   0032        movs    r2, r6"
+#  - "9e:   00be        lsls    r6, r7, #2"
```

Comment:
The fact that we're adding a dependency on the bytes being included in the output here is a bit sad, I was hoping to be able to add --no-show-raw-insn for all architectures. Oh well. An alternative would be to do real ELF parsing for this instead, like in `parse_elf_rodata_references`, but I guess it's more fragile.